### PR TITLE
feat: Add hepconvert

### PR DESCRIPTION
**Pull request by matthewfeickert**
* c.f. https://github.com/scikit-hep/hepconvert
* Recipe generated with grayskull via 'grayskull pypi hepconvert' and then edited to add additional information.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [N/A] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/hepconvert/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+about:
+  summary: File conversion package.
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - matthewfeickert
+    - zbilodea
+    - jpivarski
```
Comment (matthewfeickert) [inline recipes/hepconvert/meta.yaml:49]: @zbilodea @jpivarski Can you both comment on the PR if you would be willing to be conda-forge recipe maintainers with me for `hepconvert`?

**Comment by matthewfeickert** [comment]
@conda-forge-admin, please rerender

File: recipes/hepconvert/meta.yaml (hunk 1 above)
Comment (zbilodea) [reply recipes/hepconvert/meta.yaml:52]: That's fine with me!

Comment (matthewfeickert) [reply recipes/hepconvert/meta.yaml:52]: @zbilodea Great! The way the linter works though, it requires you to comment in the main PR thread just saying that you're willing to be a maintainer.

Comment (zbilodea) [reply recipes/hepconvert/meta.yaml:52]: I see!

**Comment by zbilodea** [comment]
I am willing to be listed as a maintainer

**Comment by jpivarski** [comment]
I am willing to be listed as a maintainer

**Comment by matthewfeickert** [comment]
@conda-forge/help-python, ready for review

**Comment by matthewfeickert** [comment]
@conda-forge-admin, please lint

**Pull request merged**